app/api-requests/application-service-api.js

```diff
@@ -1,7 +1,7 @@
 const Wreck = require('@hapi/wreck')
 const config = require('../config')
 
-async function getLatestApplicationsBySbi (sbi, logger) {
+async function getAllApplicationsBySbi (sbi, logger) {
```

Comment:
I think the problem here stems from the applications API endpoint itself. Rather than having an `/applications` endpoint, which accepts filters such as `sbi` and sorts such as `createdAt` a very specific endpoint has been created with the path `applications/latest` and calling a method which accepts a single argument `sbi` and then returns applications sorted by createdAt.

This is fine as an interim measure but what I'm saying is that we should probably address things upstream and have an endpoint more like: `/applications?sbi=123&sort=createdAt&order=desc`.

Reply:
Yeah, I don't disagree with you on that, just changed the name of this here to reflect what it is doing today. It does not return latest applications, it returns them all